product_docs/docs/bdr/4/conflicts.mdx

```diff
@@ -1013,86 +1000,86 @@ bdr.alter_node_set_conflict_resolver(node_name text,
+This function is transactional. You can roll back the changes, and
+they are visible to the current transaction.
 
-### List of Conflict Resolvers
+### List of conflict resolvers
 
-There are several conflict resolvers available in BDR, with differing coverages
+Several conflict resolvers are available in BDR, with differing coverages
 of the conflict types they can handle:
 
--   `error` - throws error and stops replication.
+-   `error` &mdash; Throws error and stops replication.
     Can be used for any conflict type.
--   `skip` - skips processing of the remote change and continues replication
+-   `skip` &mdash; Skips processing the remote change and continues replication
     with the next change.
     Can be used for `insert_exists`, `update_differing`, `update_origin_change`,
     `update_missing`, `update_recently_deleted`, `update_pkey_exists`,
     `delete_recently_updated`, `delete_missing`, `target_table_missing`,
-    `target_column_missing` and `source_column_missing` conflict types.
--   `skip_if_recently_dropped` - skip the remote change if it's for a table that
-    does not exist on downstream because it has been recently (currently within
-    1 day) dropped on the downstream; throw an error otherwise. Can be used for
+    `target_column_missing`, and `source_column_missing` conflict types.
+-   `skip_if_recently_dropped` &mdash; Skip the remote change if it's for a table that
+    doesn't exist downstream because it was recently (within
+    one day) dropped on the downstream; throw an error otherwise. Can be used for
     the `target_table_missing` conflict type. `skip_if_recently_dropped` conflict
-    resolver may pose challenges if a table with the same name is recreated shortly
-    after it's dropped. In that case, one of the nodes may see the DMLs on the
-    recreated table before it sees the DDL to recreate the table. It will then
-    incorrectly skip the remote data, assuming that the table is recently dropped
-    and cause data loss. It is hence recommended to not reuse the object namesq
+    resolver can pose challenges if a table with the same name is re-created shortly
+    after it's dropped. In that case, one of the nodes might see the DMLs on the
+    re-created table before it sees the DDL to re-create the table. It then
+    incorrectly skips the remote data, assuming that the table is recently dropped,
+    and causes data loss. We hence recommend that you don't reuse the object namesq
     immediately after they are dropped along with this conflict resolver.
--   `skip_transaction` - skips the whole transaction that has generated the
+-   `skip_transaction` &mdash; Skips the whole transaction that generated the
     conflict. Can be used for `apply_error_ddl` conflict.
--   `update_if_newer` - update if the remote row was committed later (as
+-   `update_if_newer` &mdash; Update if the remote row was committed later (as
     determined by the wall clock of the originating server) than the conflicting
```

Comment:
should we unify nomenclature between server/host/node to always be refered to as node?

Reply:
I think so. I saw your comment below and assume all instances of server can be changed to node except for in product names. I'll go ahead and make that change.

Reply:
yep thanks